topics/epigenetics/tutorials/ewas-suite/workflows/Infinium-Human-Methylation-BeadChip-tests.yml

```diff
@@ -0,0 +1,43 @@
+- doc: Test outline for Infinium-Human-Methylation-BeadChip
+  job:
+    R01C02_Red.idat:
+      class: File
+      path: test-data/R01C02_Red.idat.idat
+      filetype: idat
+    R02C02_Red.idat:
+      class: File
+      path: test-data/R02C02_Red.idat.idat
+      filetype: idat
+    R05C02_Red.idat:
+      class: File
+      path: test-data/R05C02_Red.idat.idat
+      filetype: idat
+    R06C02_Red.idat:
+      class: File
+      path: test-data/R06C02_Red.idat.idat
+      filetype: idat
+    R01C02_Green.idat:
+      class: File
+      path: test-data/R01C02_Green.idat.idat
+      filetype: idat
+    R02C02_Green.idat:
+      class: File
+      path: test-data/R02C02_Green.idat.idat
+      filetype: idat
+    R05C02_Green.idat:
+      class: File
+      path: test-data/R05C02_Green.idat.idat
+      filetype: idat
+    R06C02_Green.idat:
+      class: File
+      path: test-data/R06C02_Green.idat.idat
+      filetype: idat
+    phenotype Table txt file:
+      class: File
+      path: test-data/phenotype Table txt file.tabular
+      filetype: tabular
+    UCSC Main on Human:
+      class: File
+      path: test-data/UCSC Main on Human.gtf
+      filetype: gtf
+  outputs: {}
```

Comment:
Could you please add some output tests here? https://training.galaxyproject.org/training-material/faqs/gtn/gtn_workflow_testing.html (step 11)